paho-mqtt: redirect SRC_URI to github tree

demathif opened the pull request:
As git.eclipse.org/gitroot/paho/org.eclipse.paho.mqtt.c is nomore
available, use a github mirror.

Signed-off-by: Franck Demathieu franck.demathieu@intel.com

tripzero commented on recipes-devtools/upm/upm_0.6.1.bb:
```diff
@@ -16,6 +16,29 @@ inherit distutils-base pkgconfig python-dir cmake
+
+# include .so symlinks in main package
+FILES_${PN}_append = "${@' ${libdir}/../lib64/*.so' if '${TARGET_ARCH}' == 'x86_64' else ' ${libdir}/../lib/*.so'}"
+INSANE_SKIP_${PN} = "dev-so"
```
Can you split the java, python and nodejs files into packages like you did with mraa?  This will avoid these two lines.

andreivasiliu2211 replied:
Yes, I will split them in a future PR, but for this release there is no much time left for testing and all. I decided to include the version that was previously tested (the one that was in meta-intel-iot-devkit) + some minor changes (which include the latest UPM version). Is this acceptable for now?

tripzero replied:
It's likely this will not build in ostro (and possibly the latest poky).  I had to do this to the mraa PR before it would build:

```
+# include .so symlinks in main package
+#FILES_${PN}_append = "${@' ${libdir}/../lib64/*.so' if '${TARGET_ARCH}' == 'x86_64' else ' ${libdir}/../lib/*.so'}"
+#FILES_${PN} += "${libdir}/libmraajava.so"
+#INSANE_SKIP_${PN} = "dev-so"

+FILES_${PN} += " ${libdir}/libmraajava.so"
+FILES_SOLIBSDEV = ""
+FILES_${PN}-dev += " ${libdir}/libmraa.so"
```

Can you try this for this recipe see if it works for you?
The other alternative is to split the packages which also solves the build error.

andreivasiliu2211 commented:
I split UPM into separate packages, one for every binding (upm-java, python-upm, node-upm).

andreivasiliu2211 commented:
I split UPM into packages, one for every binding (upm-java, python-upm, node-upm). Also, there is an issue with the installation path for NodeJS files in UPM, and I solved it into a patch (see in the pull request, under upm directory). I will work with the UPM team to integrate this patch into a release, but for now, the patch file added in the recipe is the only way to solve the issue.

andreivasiliu2211 commented:
Updated this PR by not including TinyB anymore.

tripzero committed:
[tinyb] fix java architecture issue

tripzero committed:
[upm] disable python bindings until they will build consistently

Signed-off-by: Kevron Rees <tripzero.kev@gmail.com>

mufeng committed:
mosquitto: udpate to 1.4.8

Due to 1.4 version has some memory leak issue, update to 1.4.8 in hope
of solving them.

tripzero committed:
Merge pull request #1 from mufeng/uprev_mosquitto

mosquitto: udpate to 1.4.8

Commit:
paho-mqtt: redirect SRC_URI to github tree

As git.eclipse.org/gitroot/paho/org.eclipse.paho.mqtt.c is nomore
available, use a github mirror.

Signed-off-by: Franck Demathieu <franck.demathieu@intel.com>

tripzero commented:
@demathif looks good.  Can you make this PR to ostroproject/meta-intel-iot-middleware ?

demathif commented:
@tripzero sure.

EricZaluzec commented:
@tripzero , will the changes in this pull request be integrated/merged back to [git.yoctoproject.org](http://git.yoctoproject.org/cgit/cgit.cgi/meta-intel-iot-middleware/) repo for meta-intel-iot-middleware? 

Reading the iot-middleware [about](http://git.yoctoproject.org/cgit/cgit.cgi/meta-intel-iot-middleware/about/) page, I read that patches can be submitted to your github repo, but never directly mentioned when they'd be merged back to git.yoctoproject. 

If the alternative patch submission approach needs to be used (submit to the meta-intel mailing list
(meta-intel@yoctoproject.org)) to get these changes along, let me know and I'd be happy to start that process! For the community!

Thanks.
